### DISCUSSION TITLE
Suggestions from `cmp-nvim-lsp` are super weird

### DISCUSSION THREAD

**Discussion by teenjuna** [hrsh7th/nvim-cmp#794] [Q&A] [answered]
Hi! I've set up the `nvim-cmp` with a couple of sources ([related lines in config](https://github.com/teenjuna/configs/blob/master/nvim/lua/plugins/nvim-cmp.lua#L24-L45)), but the `cmp-nvim-lsp` source produces something weird.

For example:

<img width="472" alt="Screenshot 2022-02-13 at 12 17 13" src="https://user-images.githubusercontent.com/53595243/153739929-0ed6a56e-40de-4b9c-a08c-43053b32b765.png">

If I comment out `nvim_lsp` source, I have this:

<img width="472" alt="Screenshot 2022-02-13 at 12 17 57" src="https://user-images.githubusercontent.com/53595243/153739978-266410c0-933f-42fa-9bd1-f7d5ceeb0d44.png">

Which is much more usable, since `nvim_lsp` source just filled my suggestions with some weird random garbage and prevented `buffer` source from appearing (it has lower priority).

I would be very thankful if someone could show me how to fix this behavior (basically don't show random `Text` garbage from `nvim_lsp` source).

PS: this issue happens not only in Lua file. Here is HTML example:

<img width="210" alt="Screenshot 2022-02-13 at 12 28 52" src="https://user-images.githubusercontent.com/53595243/153740201-90787124-908b-4fa4-8376-abf59ecc256e.png">

**Comment by teenjuna** [comment]
I should note that I'm using neovim nightly (last pull was today) and latest nvim-cmp.

**Comment by hrsh7th** [comment, marked as answer]
I don't know. But I'm worried about the null-ls completion. Could you test without null-ls?

**Reply by teenjuna** [reply]
You're right, thank you. This is solved by commenting [this line](https://github.com/teenjuna/configs/blob/162aa4b4e0d3fdb0b2b9f709fab89a70ed7be72e/nvim/lua/plugins/lsp.lua#L89) out. I don't even remember how it got there...

**Reply by rstacruz** [reply]
Thank you so much for this! This happened with my setup too. This line was suggested in the docs of null-ls.
